TestFiles/bad.json

```diff
@@ -1,72 +1,840 @@
 [
```

Comment:
For these test files, while it is more readable to pretty print the test lines, it'll be inconsistent with the other test files. That said, I'm not too sure on how these test files should be structured and will leave it up for anyone more experience to leave a comment.

Reply:
Thanks, will coordinate with other repos

Reply:
done